unstructured/ingest/connector/astra.py

```diff
@@ -35,6 +34,8 @@ class SimpleAstraConfig(BaseConnectorConfig):
     access_config: AstraAccessConfig
     collection_name: str
     embedding_dimension: int
+    namespace: t.Optional[str] = None
```

Comment:
if you want them exposed to the cli, namespace and requested_indexing_policy need to be added to 

https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/cli/cmds/astra.py

Reply:
Done! i tried to match the data model to the "required" option for the CLI, hopefully this looks good.